Makefile

```diff
@@ -112,10 +93,10 @@ deploy-elasticsearch-operator:
 deploy-example: deploy
 	oc create -n $(NAMESPACE) -f hack/cr.yaml
 
-test-unit: fmt
-	@LOGGING_SHARE_DIR=$(CURDIR)/files go test $(TEST_OPTIONS) $(PKGS)
+test-unit: generate
+	go test ./pkg/...
 
-test-e2e:
+test-e2e: build
```

Comment:
We should rethink the dependency upon `build` as its unnecessary.  In a CI context, the operator will be deployed using the image that was built by a separate CI job.  Locally the deployment will reference an image that only exists if you run `make deploy-image`.  This means the primary place this  is used we are building the operator twice.

Reply:
+1 I only left it there because I assumed it was a CI requirement. Will remove.